Team-Charter.md

```diff
@@ -0,0 +1,132 @@
+# Vue.js Team Guidelines
+
+## Introduction
+
+As an open-source project, Vue.js supports the community by giving them confidence that the team will consistently maintain, drive, and guide the project. These guidelines outline the expectations of Vue.js team members for current members and those who would like to be on the team.
+
+## Background
+
+_This section was inspired by the[ Open Source Guide](https://opensource.guide/leadership-and-governance/#what-are-examples-of-formal-roles-used-in-open-source-projects)._
+
+There are three roles that people often take on when it comes to contributing to open source:
+
+- **Maintainer** - People who feel responsible for the direction of the project and is committed to improving it
+- **Contributor** - People who add value to the project (i.e., triaging issues, pull requests, updating documentation, etc.)
+- **Committer** - People who have commit and/or release permissions
+
+Vue.js team members take on these roles to varying degrees based on the individual. In addition, there is one additional trait that is critical to being a team member: _consistency**.**_
+
+## What is the Vue.js Team?
+
+The Vue.js Team consists of members in the community who have provided valuable contributions to the community and are recognized for their efforts and time. While contribution differs between individuals, each person contributes to the ecosystem in their own individual way.
+
+### What is expected of a team member?
+
+#### #1. Maintain a consistent presence in the project
```

Comment:
Small nit: now that I see it in here, do we need the numbers in the headings?

Reply:
Yeah good point. I don't think we need it since we only have two points. If we had three or more, maybe it'd be worth it.